controllers/api/v1/health_check_test.go

```diff
@@ -1,41 +1,94 @@
 package apiv1
 
 import (
+	"fmt"
 	"io/ioutil"
 	"net/http"
 
+	_ "google-scraper/initializers"
 	. "google-scraper/tests"
 
+	"github.com/google/uuid"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("HealthCheckController", func() {
 	Describe("GET /api/v1/health-check", func() {
-		It("returns 200 status code", func() {
-			response := MakeRequest("GET", "/api/v1/health-check", nil)
+		Context("Given the valid credential", func() {
```

Comment:
From @olivierobert comment, `the` is redundant here. Please update this context and also another context.